.azure-pipelines/client.test.yml

```diff
@@ -38,7 +35,7 @@ jobs:
       - task: Npm@1
         inputs:
           command: 'custom'
-          customCommand: 'run mocha -- $(build.path) --reporter mocha-junit-reporter'
+          customCommand: 'run mocha -- $(test.path) --reporter mocha-junit-reporter'
```

Comment:
Where is `test.path` defined? Is this an azure-devops thing?

Reply:
Correct, `test.path` and `source.path` are variables supplied by the DevOps pipeline.